parser/parser_references.go

```diff
@@ -43,13 +43,14 @@ func (r *resourceUsageVisitor) Visit(cursor *walker.Cursor) (w walker.Visitor) {
 		resource, parser := r.resourceAndFuncFor(node)
 		if parser != nil {
 			if parser.AllowedLocations.Allowed(cursor.Location()) {
-				parser.Parse(r.p, r.file, resource, node)
+				parser.Parse(r.p, r.file, resource, cursor, node)
 			} else {
 				call := fmt.Sprintf("`%s.%s`", resource.Ident().Name, parser.Name)
 				r.p.errf(node.Pos(),
-					"You can not call %s here, %s️.",
+					"You can not call %s here, %s️. For more information please see %s",
```

Comment:
"please" sounds a bit strange in a compiler error. Also "cannot" instead of "can not". How about:
```suggestion
					"You cannot call %s here, %s️. For more information see %s",
```